src/services/SaveService.ts

```diff
@@ -553,6 +554,7 @@ export class SaveService {
       p.dropCapDefaultFontStyle ?? pageSetup.dropCapDefaultFontStyle;
     pageSetup.dropCapDefaultStrokeWidth =
       p.dropCapDefaultStrokeWidth ?? pageSetup.dropCapDefaultStrokeWidth;
+    pageSetup.dropCapDefaultLineHeight = p.dropCapDefaultLineHeight;
```

Comment:
```suggestion
    pageSetup.dropCapDefaultLineHeight = p.dropCapDefaultLineHeight ?? pageSetup.dropCapDefaultLineHeight;
```

It's best to fall back here to the default page setup line height to keep the existing code pattern.